sdk/ml/azure-ai-ml/azure/ai/ml/_schema/component/input_output.py

```diff
@@ -57,7 +58,9 @@ class PrimitiveOutputSchema(OutputPortSchema):
         required=True,
     )
     is_control = fields.Bool()
```

Comment:
may also hide is_control?

Reply:
sure; this change may break some tests for schema issue, handling them may take some time (add fixture here and there)